### PR TITLE
containerd_nerdctl: Add support for aarch64

### PR DESCRIPTION
**Pull request by ggardet**
- Verification run on TW: 
  - aarch64: https://openqa.opensuse.org/t2463204
  - x86_64: https://openqa.opensuse.org/t2463199

### CODE REVIEW

File: lib/containers/containerd_nerdctl.pm [hunk 1]
```diff
@@ -11,15 +11,22 @@ use Mojo::Base 'containers::engine';
...
 has runtime => 'nerdctl';
 
 sub init {
     install_containerd_when_needed();
 
     # The nerdctl validation test suite is a plug-in required for this test and needs to be installed from an external source.
     my $version = get_var('CONTAINERS_NERDCTL_VERSION', '0.16.1');
-    my $url = "https://github.com/containerd/nerdctl/releases/download/v$version/nerdctl-$version-linux-amd64.tar.gz";
-    my $filename = "/tmp/nerdctl-$version-linux-amd64.tar.gz";
+    my $arch = 'amd64';
```
Comment (jlausuch) [inline lib/containers/containerd_nerdctl.pm:22]: Another possible implementation:
```
my $arch;
if (is_aarch64) {
    $arch = 'arm64';
} elsif (is_x86_64) {
    $arch = 'amd64';
} else {
    die 'Architecture ' . get_required_var('ARCH') . ' is not supported';
}
```

**Review by jlausuch — APPROVED** [review]
Nice to enable this on arm! I have a suggestion, but feel free to ignore as it's pure personal taste. LGTM anyways :)

File: lib/containers/containerd_nerdctl.pm [hunk 2]
```diff
@@ -11,15 +11,22 @@ use Mojo::Base 'containers::engine';
...
+    if (is_aarch64) {
+        $arch = 'arm64';
+    } elsif (!is_x86_64) {
+        die 'Architecture ' . get_var('ARCH') . ' is not supported';
+    }
```
Comment (b10n1k) [inline lib/containers/containerd_nerdctl.pm:27]:
```
my $arch = 'arm64' if (is_x86_64 or is_aarch64);
die 'Architecture ' . get_required_var('ARCH') . ' is not supported' unless $arch
```

Comment (jlausuch) [reply lib/containers/containerd_nerdctl.pm:27]: Can't. It should be `amd64` for x86... not `arm64` :)

File: lib/containers/containerd_nerdctl.pm (hunk 1 above)
Comment (ggardet) [reply lib/containers/containerd_nerdctl.pm]: Done.

**Comment by ggardet** [comment]
New VRs are running.

**Pull request merged**